docusaurus/docs/iOS/guides/push-notifications.md

```diff
@@ -1,3 +1,104 @@
 ---
 title: Push Notifications
----
\ No newline at end of file
+---
+
+# Push Notifications
+
+First step on a way to setting up push notifications is authentication.
+Stream supports both **Certificate-based provider connection trust (.p12 certificate)** and **Token-based provider connection trust (JWT)**. Token based authentication is the preferred way to setup push notifications. This method is easy to setup and provides strong security. You can find more on setting up authentication [here](https://getstream.io/chat/docs/php/push_ios/?language=swift).
```

Comment:
We should put much more emphasis there, to make sure whoever reads this guide cannot skip over this paragraph. Maybe wrap it in `:::warning`?

Reply:
Warnings look really scary, what do you think if we instead use `caution`?

Reply:
works 👍